.github/workflows/semgrep-rules-test.yml

```diff
@@ -18,7 +18,15 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           python3 -m pip install semgrep
+          python3 -m pip install jsonschema pyyaml
       - name: validations
         run: semgrep --validate --config .
       - name: tests
         run: semgrep --test --test-ignore-todo
+      - name: metadata-tests
+        run: |
+          wget https://raw.githubusercontent.com/returntocorp/semgrep-rules/c5ad4bb0f4c7ee5b8cd47276b582e8bb57bd0a4d/.github/scripts/validate-metadata.py
+          wget https://raw.githubusercontent.com/returntocorp/semgrep-rules/develop/metadata-schema.yaml.schm
+          python ./validate-metadata.py -s ./metadata-schema.yaml.schm -f .
+      - name: x
```

Comment:
Can we name this something like "run semgrep yaml rules"

Reply:
łeło, good catch